packager/react-packager/src/Bundler/index.js

```diff
@@ -196,16 +196,15 @@ class Bundler {
       throw new Error('No matching project root for ', path);
     }
 
+    // Replaces '\' with '/' for Windows paths.
+    path = path.replace(/\\/g, '/');
```

Comment:
Here the generated URLs had '\' in them instead of '/' and caused an error when downloading source maps.